packages/design-system-scripts/gulp/docs/generatePages/generateHtmlExample.js

```diff
@@ -57,4 +59,19 @@ function generateHtmlExample(page, modifier, docsPath, options) {
   );
 }
 
+/**
+ * Blast Analytics code to be included in the <head>.
+ * This loads additional tracking scripts, like Google Analytics.
+ * @return {String}
+ */
+function analytics() {
```

Comment:
Small suggestion, since this is identical to the code in `generateDocPage`, can we just import this function from that file?